dt/uni-alpha/kustomization.yaml

```diff
@@ -0,0 +1,190 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+transformers:
+  - |-
+      apiVersion: builtin
+      kind: NamespaceTransformer
+      metadata:
+        name: _ignored_
+        namespace: openstack
+      setRoleBindingSubjects: none
+      unsetOnly: true
+      fieldSpecs:
+        - path: metadata/name
+          kind: Namespace
+          create: true
+
+components:
+  - ../../lib/networking
+  - ../../lib/control-plane
+
+replacements:
+  - source:
+      kind: ConfigMap
+      name: service-values
+      fieldPath: data.cinderVolumes.customServiceConfig
+    targets:
+      - select:
+          kind: OpenStackControlPlane
+        fieldPaths:
+          - spec.cinder.template.cinderVolumes.customServiceConfig
```

Comment:
As I explain in a later comment, the CinderVolumes CRD is a list/map of backends. Assuming that you choose the same "lvm-iscsi" name, the actual field path will be "spec.cinder.template.cinderVolumes.lvm-iscsi.customServiceConfig"

If possible, you should consider replacing the entire spec.cinder.template.cinderVolumes, not just the backend's customServiceConfig. As I note below, you'll also need to specify a nodeSelector so that the pod is scheduled on the same node where the LVM volume data is stored.

Reply:
Done.